src/styles/card.scss

```diff
@@ -0,0 +1,105 @@
+$secondary: #616070;
+$btn-white-color: #fff;
+$btn-turquoise-color: #00acdc;
+$main-acent: #060b35;
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &photo {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $main-acent;
+    margin-bottom: 4px;
+    margin-top: 0;
+  }
+
+  &code {
+    line-height: 14px;
+    text-align: left;
+    margin-bottom: 16px;
+    margin-top: 0;
+    color: $secondary;
+  }
+
+  &price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 16px;
+    margin-top: 0;
+    color: $secondary;
+  }
+
+  &price-value {
+    font-size: 16px;
+    font-weight: 700;
+    text-align: right;
+    color: #060b35;
+  }
+
+  &price-header {
+    font-size: 12px;
+    text-align: left;
+    color: #616070;
+  }
+
+  &review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    margin-top: 0;
+    height: 16px;
+    width: 166px;
+  }
+
+  &review-count {
+    text-align: right;
+    color: $main-acent;
+  }
+
+  &__btn {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    color: $btn-white-color;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    width: 100%;
+    height: 40px;
+    box-sizing: border-box;
+    border: 1px solid $btn-turquoise-color;
+    border-radius: 5px;
+    background: $btn-turquoise-color;
+
+    &:hover {
+      background: #fff;
+      color: $btn-turquoise-color;
+    }
+  }
```

Comment:
The &__btn seems to be a modifier, but it's written as if it's an element. If it's a modifier, it should follow the BEM naming convention with a single underscore and a descriptive name, like &__btn--primary. If it's an element, it should be named &__btn.